modules/ROOT/pages/configuration-schema-database.adoc

```diff
@@ -21,6 +21,9 @@ include::partial$block-caveats.adoc[tags="legacy-config-equivalents"]
 From {sgw-te} 3.0 you can use the Admin REST API to provision persistent configuration changes.
 This content introduces the <<lbl-create-db>> and <<lbl-configure-db>> endpoints for convenience -- see {rest-api-admin--xref--database-configuration} for a full description of the endpoints available.
 
+NOTE: You can define 1 custom scope per database with up to 1000 custom collections in release 3.1. +
+Scopes and collections are optional. 
+Default scopes and collections will be applied automatically if non are specified.
```

Comment:
I think Iveta has already queried this wording.
Also typo on "none".

You'll want to mention "the default scope and collection" (e.g. there's only 1 default scope/collection, not plural). And perhaps just "used" or "assigned" instead of "applied"?